### DISCUSSION TITLE
Recommend registering Galaxy training workflows with Dockstore

### DISCUSSION THREAD

**Discussion by bvizzier-ucsc** [galaxyproject/training-material#2735] [Ideas]
In July 2021 [Dockstore](https://dockstore.org) released ["Launch with Galaxy" functionality](https://docs.dockstore.org/en/stable/getting-started/getting-started-with-galaxy.html).  

We encourage training workflows be registered with Dockstore for reference by people and as a way to see how to use Dockstore with the platform. 

Please let me know if the Dockstore team can help.

Ben
bvizzier@ucsc.edu

**Comment by bgruening** [comment]
@bvizzier thanks a lot!  Galaxy integration in Dockstore is awesome.

Do you want really every workfkows in Dockstore? Or should they be curated?

**Reply by bvizzier-ucsc** [reply]
Which workflows get registered is up to the Galaxy team.  I would suggest key ones that are good examples for users. 

@bethsheets may be able to provide some guidance.  One of the responsibilities of her team is to help with Dockstore outreach and training.

**Reply by bethsheets** [reply]
@bgruening We are hoping to improve Galaxy resources on Dockstore and help the Galaxy community learn about Dockstore and how it may be useful to them. I think curated is a great place to start and I saw you already started an org on Dockstre (https://dockstore.org/organizations/iwc). Do you have example workflows that are helpful for learning the Galaxy language that could be shared? 

Also, could we help provide a "How to publish your Galaxy workflow in Dockstore" document in your Galaxy documentation (https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/workflow-editor/tutorial.html)?

**Reply by hexylena** [reply]
@bethsheets we'd love to have documentation from your side that we could include in the workflow tutorial series, that'd be fantastic.

**Reply by bethsheets** [reply]
@hexylena Great! We are updating our own documentation and then will do a PR against your doc repo to add a tutorial for using Dockstore to find and launch Galaxy workflows. 

Do you have a suggested spot in your repo where a Dockstore tutorial would fit best? Maybe in your intro materials? https://github.com/galaxyproject/training-material/tree/main/topics/introduction

**Reply by hexylena** [reply]
We don't have a great series on workflows yet but we're working on it in the https://github.com/galaxyproject/training-material/tree/main/topics/galaxy-interface/tutorials folder, maybe as a standalone tutorial since we encourage small/micro tutorials as well.
